tests/foreman/maintain/test_upgrade.py

```diff
@@ -211,3 +212,95 @@ def test_positive_check_presence_satellite_or_satellite_capsule(sat_maintain):
 
     :CaseAutomation: ManualOnly
     """
+
+
+@pytest.mark.upgrade
+def test_positive_upgrade_check_capsule_hammer(module_capsule_configured):
+    """Test satellite-maintain upgrade check for capsule with hammer installed.
+
+    :id: 903a4140-74f3-428a-8768-ecb610fef2fa
+
+    :steps:
+        1. Sync utils repo on Capsule
+        2. install hammer on Capsule
+        3. Add satellite-capsule repo on Capsule
+        4. Add maintenance repo on Capsule
+        5. run satellite-maintain upgrade check
+        6. run satellite-maintain upgrade run
+        7. verify that the capsule is upgraded
+        8. verify that the hammer is installed on Capsule
+
+    :expectedresults:
+        1. The utils repo should be installed on Capsule
+        2. The hammer should be installed on Capsule
+        3. The satellite-capsule repo should be installed on Capsule
+        4. The capsule should be upgraded
+        5. The hammer should be installed on Capsule
+
+    :Verifies: SAT-37909
+    """
+    repo_config = f"[utils]\nname=Utils Repository\nbaseurl={settings.repos.satutils_repo}\nenabled=1\ngpgcheck=0\n"
+    result = module_capsule_configured.execute(
+        f'cat > /etc/yum.repos.d/utils.repo << EOF\n{repo_config}\nEOF'
+    )
+    assert result.status == 0
+    result = module_capsule_configured.execute('yum clean all && yum makecache')
+    assert "Utils Repository" in result.stdout
+    assert result.status == 0
+    result = module_capsule_configured.execute('yum install rubygem-hammer_cli_katello -y')
+    assert result.status == 0
+    result = module_capsule_configured.execute('rpm -qa | grep -i "hammer"')
+    assert "hammer_cli_katello" in result.stdout
+    assert result.status == 0
+
+    capsule = dogfood_repository(
+        ohsnap=settings.ohsnap,
+        repo='capsule',
+        product='capsule',
+        release='stream',
+        os_release=9,
```

Comment:
Here we could use [os_version](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/hosts.py#L303) (`os_release=module_capsule_configured.os_version.major`) so the test becomes future-proof when SAT and CAPS are based on RHEL10+.